static/js/app.ts

```diff
@@ -119,10 +120,11 @@ export interface InitializeAppOptions {
  */
 export function initializeApp(options: InitializeAppOptions) {
   theLevel = options.level;
+  theKeywordLanguage = options.keywordLanguage;
   initializeSyntaxHighlighter({
     keywordLanguage: options.keywordLanguage,
   });
-  initializeHighlightedCodeBlocks(options.keywordLanguage);
+  initializeHighlightedCodeBlocks(document.body);
```

Comment:
This might require a bit of an explanation (either here or in the docs) the old version clearly explains what we are fetching but what is in the body and why do we use it here?